### PR TITLE
Adding Dell Gen14 support

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
As per issues reported in #25 , let's add a specific test suite for C6420.

### CODE REVIEW

File: hwbench/environment/tests/vendors/Dell/C6420/oem_system [hunk 1]
```diff
@@ -0,0 +1,907 @@
+{
```
Comment (anisse) [inline hwbench/environment/tests/vendors/Dell/C6420/oem_system:1]: How do you generate the test files to be in python format and not json ?

**Review by anisse — COMMENTED** [review]
I noticed small typos in the last commit message:
will complains -> will complain
paramater -> parameter

Otherwise LGTM!

File: hwbench/environment/tests/vendors/Dell/C6420/oem_system (hunk 1 above)
Comment (ErwanAliasr1) [reply hwbench/environment/tests/vendors/Dell/C6420/oem_system:1]: that's a curl output, slightly modified:

- null -> None
- false -> False
- true -> True

Comment (anisse) [reply hwbench/environment/tests/vendors/Dell/C6420/oem_system:1]: This makes me wonder if we shouldn't change this to use json directly in the future. Do you remember why we picked python ast over json to load test files?

**Review by anisse — APPROVED** [review]

**Pull request merged**